<launch>
	<node name="data_processor" pkg="drone" type="data_processor" respawn="false" output="screen">
		<param name="client_configuration" type="string" value="$(find drone)/config/client.config"/>
		<param name="server_configuration" type="string" value="$(find drone)/config/server.config"/>
	</node>
</launch>
